### DISCUSSION TITLE
App freezing when navigating between collections

### DISCUSSION THREAD

**Discussion by jd-isc** [directus/directus#5587] [Archive]
We have recently implemented Directus as a CMS for our project. We have run into an issue - switching between the collections in Directus is causing the app to freeze for us. After initially logging in, we are usually able to navigate between collections fine, but after a couple clicks, we see a 'loading...' message. This message stays there until we either log out and back in or sometimes it resolves when we go to a different section in the module menu and then back to collections. This occurs as an admin and as other users with less permissions and it happens with collections that have very few records. 

Has this been encountered before and if so, is there a setting/workaround that we can implement?

**Comment by rijkvanzanten** [comment]
@jd-isc What version of Directus are you on? What layouts are you using in these collections? What options are you using to view this data? Would you be able to share a DB dump with a minimal reproduction we can use to debug?

**Reply by jd-isc** [reply]
@rijkvanzanten, we are using v9.0.0-rc.64. 

I did some more testing and it looks like the problem is with one collection in particular. If I navigate between all the other collections multiple times, I don't see the freezing. But as soon as I click on the collection in question, it shows the loading screen. Then when I try to navigate away to other collections, they are all then stuck on the loading screen. I've removed all presets/bookmarks for this collection with no change. I've also removed any sorting from within the data model settings with no change.

For the DB dump, could you tell me what you mean by 'minimal reproduction'?

**Reply by jd-isc** [reply]
@rijkvanzanten 

We have a collection named 'info screen' and another named 'info section'. The 'info section' collection is the one giving us issues. 

It’s interesting, the log shows for info screens doing select to get screen data then returning it via web response then saving info screens as “last page” (viewed).
Then the only thing showing for info section is saving as “last page” (viewed). It doesn’t look like it even tried a select statement (and therefore no web response)

[Directus-logs-20210511-100019.xlsx](https://github.com/directus/directus/files/6460711/Directus-logs-20210511-100019.xlsx)

**Reply by rijkvanzanten** [reply]
> For the DB dump, could you tell me what you mean by 'minimal reproduction'?

Everything I might possibly need to see the issue for myself locally 🙂 From your messages above, it sounds like there's a specific collection that's causing issues, so a DB dump that includes that would be very useful

**Reply by jd-isc** [reply]
Okay, is there a way we can share the DB dump with you without posting it publicly on github?

**Reply by rijkvanzanten** [reply]
Totally! You can send it directly to me through email: rijk@directus.io 👍🏻
